lib/lbt/bundle/BundleDefinition.js

```diff
@@ -18,7 +18,13 @@ const SectionType = {
 	Preload: "preload",
 
 	/**
-	 * For each module, a jQuery.sap.require call will be created.
+	 * Content information for another bundle is written.
+	 * Requires UI5 Evolution runtime.
```

Comment:
As discussed with Sven: What version does this refer to?

Reply:
Answer: https://github.com/SAP/openui5/commit/fb4dbd91f2dac6bcce276a7eefbd8d42e8cc044b has been added with 1.74.0